src/changelogs/fetch_changelogs.py

```diff
@@ -19,78 +19,70 @@ class Changelog(TypedDict):
     link: str
 
 
-class ChangelogLine(TypedDict):
-    """Each changelog line in a <changelog_id>.json file"""
+class ChangelogString(TypedDict):
+    """Each complete changelog in a <changelog_id>.json file"""
 
-    Description: str
-    Tags: list[str]
+    changelog_string: str
 
 
 class ChangelogFetcher:
     """
-    Fetches changelogs from the deadlock forums and parses them into a dictionary
+    Fetches changelogs from the deadlock forums and game files and parses them into a dictionary
     """
 
-    # Hero lab changelogs need to be added manually to
-    # ./input-data/raw-changelogs following the
-    # naming convention 'herolab_2024_10_29.txt'
-    # see the referenced file for example formatting
+    def __init__(self, client_version, update_existing):
+        self.changelogs: dict[str, ChangelogString] = {}
+        self.changelog_configs: dict[str, ChangelogConfig] = {}
+        self.client_version = client_version
```

Comment:
this is unused

Reply:
Ignore `client_version` for now, it's in preparation of manifest-id versioning. Consider it a reminder for our future selves :wink:

Reply:
I know you're half joking, but if its a necessary part of the manifest feature, create a git issue with that information instead of unused code

Reply:
Aye sounds good. If I'm being serious, I had added it pre-emptively and forgotten that it would actually require the manifest-versioning branch until you mentioned it here lol.